examples/users_exmaple.pp

```diff
@@ -0,0 +1,8 @@
+registry_key { "HKU\\${windows_currentuser_sid}\\Test2":
+  ensure => present
+}->

```

Comment:
@joshcooper Small world, indeed.  Just don't laugh at my ruby code, i'm still getting over my python bias :)

yeah, i started to go down the path of having the module do a SID lookup locally via WMI, but pulled back, and thus i should have removed the param.  

I'm happy to try and get the SID supplied via params (username, domain), but I know best practice is to usually use facts where possible. Advice ?